src/Microsoft.ML.Transforms/Text/StopWordsRemovingTransformer.cs

```diff
@@ -1076,10 +1086,25 @@ protected override Delegate MakeGetter(DataViewRow input, int iinfo, Func<int, b
     }
 
     /// <summary>
-    /// Custom stopword remover removes specified list of stop words.
-    /// This is usually applied after tokenizing text, so it compares individual tokens
-    /// (case-insensitive comparison) to the stopwords.
+    /// <see cref="IEstimator{TTransformer}"/> for the <see cref="CustomStopWordsRemovingTransformer"/>.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | Vector of [Text]<xref:System.ReadOnlyMemory{System.Char}> |
+    /// | Output column data type | Vector of unknown size of [Text]<xref:System.ReadOnlyMemory{System.Char}> |
+    ///
+    /// The resulting <xref:Microsoft.ML.Transforms.Text.CustomStopWordsRemovingTransformer/> creates a new column, named as specified in the output column name parameters, and
+    /// fills it with vector of texts similar to vector of texts in input column but removing all specified texts from it.
```

Comment:
specified texts -> common words?

Reply:
vector of texts -> vector of words?

Reply:
they not common, they provided by user. does that phrasing looks better?

---
In reply to: [276881001](https://github.com/dotnet/machinelearning/pull/3413#discussion_r276881001) [](ancestors = 276881001)